lib/index.ts

```diff
@@ -7,24 +7,39 @@ import { Docker, DockerOptions } from './docker';
 import { buildResponse } from './response-builder';
 
 export {
+  EnvDetails,
   inspect, dockerFile,
 };
 
+// Additional details which can help the analyzer
+// make a decision
+//
+interface EnvDetails {
```

Comment:
That might be over-engineering it a bit but I think it is better to have an environment object rather then passing the dockerfile details directly around.

Reply:
I do feel like this might be over-engineered, but beyond that,  "EnvDetails" does not really explain what am I expecting to find in it. I'd start with the least common denominator rather than jumping to generic terms, if we ever find a more elaborate use to it, we can readdress the name

Reply:
For starters, perhaps I would actually pass dockerfile analysis.